v-next/hardhat/templates/empty-javascript/package.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "empty-javascript",
+  "description": "An empty JavaScript Hardhat project",
+  "type": "module",
+  "devDependencies": {
+    "@ignored/hardhat-vnext": "2.0.0"
```

Why do some of these tempaltes have `workspace:` dependencies and others don't?

---

I wanted to test the path with `hardhat --init` actually installing the dependencies from `npm`. 

The way I propose handling `workspace:` deps from the templates is to replace them with whatever `hardhat` version we're running the `--init` from. That way, we won't have to update the hardhat version in template package.json(s) all the time. At the same time, we'll be able to test the templates themselves as they are and where they are with a workspace version of `hardhat`.

The one downside is that when we're running `--init` from an unreleased version of `hardhat`, we cannot just install the project deps from `npm` because they are not there yet. If we wanted to handle that better, maybe we could introduce some test flag/env var for that?

---

Having said that, I think we should reference the workspace packages in the template `package.json`s.

```suggestion
    "@ignored/hardhat-vnext": "workspace:^3.0.0-next.3"
```